Fish.cs

```diff
@@ -0,0 +1,38 @@
+using Godot;
+using System;
+
+public partial class Fish : Area2D
+{
+	public float BaseSpeed { get; set; } = 200;
```

Comment:
Set accessors are not consistent.
Please revise and group/sort code order by access

Reply:
fixed